Detector/DetFCChhBaseline1/compact/FCChh_DectDimensions.xml

```diff
@@ -65,58 +65,55 @@
     <constant name="FwdTracker_z2" value="2000*mm"/>
     <constant name="FwdTracker_zOffset" value="14000*mm"/>
```

Comment:
Hi,

the FwdHelp (intermediate forward tracker between the central tracker and forward tracker) is described here: http://fcc-tklayout.web.cern.ch/fcc-tklayout/FCChh_Option3.v02/indexFwdHelp.html
The parameters are: rMin = 40mm rMax = 1189.4, zMean = 7500

The Fwd tracker can be in detail found here: http://fcc-tklayout.web.cern.ch/fcc-tklayout/FCChh_Option3.v02/indexFwd.html